app/controllers/catalog_controller.rb

```diff
@@ -4,8 +4,11 @@ class CatalogController < ApplicationController
   include Blacklight::Catalog
 
   configure_blacklight do |config|
-          config.show.oembed_field = :oembed_url_ssm
-          config.show.partials.insert(1, :oembed)
+  config.show.oembed_field = :oembed_url_ssm
```

Comment:
Should this indention change have 2 additional spaces? Or is the GitHub diff messed up.

Reply:
I'm so confused about indentation in that file.

Reply:
yeah.. #itsComplicated